hoi/plot/landscape.py

```diff
@@ -59,33 +67,33 @@ def plot_landscape(hoi, model=None, orders=None, minsize=None, maxsize=None,
     orders = orders[keep]
 
     # switch between histogram and scatter plot
-    if kind == 'hist':
-        hist_kwargs['output'] = 'numpy'
+    if kind == "hist":
+        hist_kwargs["output"] = "numpy"
         lscp, order, bins = landscape(hoi, orders, **hist_kwargs)
 
-        plt.pcolormesh(
-            order, bins, lscp, norm=LogNorm(), **plt_kwargs)
+        plt.pcolormesh(order, bins, lscp, norm=LogNorm(), **plt_kwargs)
         if cbar:
-            plt.colorbar(label=hist_kwargs.get('stat', 'probability'))
-    elif kind == 'scatter':
+            plt.colorbar(label=hist_kwargs.get("stat", "probability"))
+    elif kind == "scatter":
         size = normalize(np.abs(hoi), to_min=2, to_max=200)
         minmax = abs(np.nanpercentile(hoi, [1, 99])).max()
         for o in range(minsize, maxsize + 1):
             keep = orders == o
             hoi_o = hoi[keep]
             x = np.random.normal(loc=o, scale=0.13, size=hoi_o.size)
-            plt.scatter(x, hoi_o, c=hoi_o, vmin=-minmax, vmax=minmax,
-                        s=size[keep], **plt_kwargs)
+            n = minmax
+            t = plt_kwargs
+            plt.scatter(x, hoi_o, c=hoi_o, vmin=-n, vmax=n, s=size[keep], **t)
```

Comment:
I don't understand why you had to change this line, it was on two lines. Now it looks a bit weird this `n = minmax` and `t = plt_kwargs`